
FAIL respondWith() takes either a Response or a promise that resolves with a Response. Other values should raise a network error. promise_test: Unhandled rejection with value: object "Error: wait_for_state must be passed a ServiceWorker"

